skype/skype-ps/skype/Get-CsTeamsAppPermissionPolicy.md

```diff
@@ -38,7 +38,45 @@ As an admin, you can use app permission policies to enable or block specific app
 ## EXAMPLES
 
 ### Example 1
-Intentionally omitted.
+
+Get the Global Teams App Permission Policy.
+```powershell
+Get-CsTeamsAppPermissionPolicy -Identity Global
+
+Identity               : Global
+DefaultCatalogApps     : {Id=26bc2873-6023-480c-a11b-76b66605ce8c, Id=0d820ecd-def2-4297-adad-78056cde7c78, Id=com.microsoft.teamspace.tab.planner}
+GlobalCatalogApps      : {}
+PrivateCatalogApps     : {}
+Description            :
+DefaultCatalogAppsType : AllowedAppList
+GlobalCatalogAppsType  : AllowedAppList
+PrivateCatalogAppsType : AllowedAppList

```

Comment:
Please use the output format:
https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide#executable-examples

Thanks!